packages/widgets-mitosis/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "widgets-mitosis",
```

Comment:
I recommend you add these widgets to the SDKs workspaces:

https://github.com/BuilderIO/builder/blob/c82d2bb976a9d9c3421dc98e0b0233d568566a8f/packages/package.json#L5-L11

This will make development easier if you need to import the SDKs or add tests for the widgets in the future.

Reply:
when i add the widgets-mitosis to workspaces, the build script fails, how to change it to make it work again?

Reply:
I need more info about the error to be able to help you